### PR TITLE
Fixes #13971: Remove windows reference in rudder techniques

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/13971

### CODE REVIEW

**Comment by peckpeck** [comment]
Commit modified

File: techniques/system/common/1.0/cf-serverd.st [hunk 1]
```diff
@@ -93,11 +93,6 @@ bundle server access_rules
 
     # Allow server to remotely run the agent
     # Using ${sys.cf_agent} fails, as cf-serverd cannot canonize its path
-    windows::
-      "${sys.workdir}\bin\cf-agent.exe"
-        admit_ips => { host2ip("${server_info.policy_server}") };
-
-    !windows::
```
Comment (amousset) [inline techniques/system/common/1.0/cf-serverd.st]: I think we need an `any::` here

File: techniques/system/common/1.0/promises.st [hunk 2]
```diff
@@ -25,13 +25,8 @@ bundle common rudder_roles
 {
   vars:
     # moved from bundle common g since they need to be defined in promises.cf
-    android::
-      "uuid_file"         string => "/data/rudder/etc/uuid.hive";
-    windows::
-      "uuid_file"         string => "${sys.winprogdir}\Rudder\etc\uuid.hive";
-    !windows.!android::
-      "uuid_file"         string => "/opt/rudder/etc/uuid.hive";
     any::
```
Comment (amousset) [inline techniques/system/common/1.0/promises.st]: useless

File: techniques/systemSettings/process/servicesManagement/3.0/servicesManagement.st [hunk 3]
```diff
@@ -104,56 +104,9 @@ bundle agent services_management
       "any" usebundle => rudder_common_report_index("ServicesManagement", "result_success", "${service[${index}][uuid]}", "Process", "${service[${index}][name]}", "${service[${index}][name]} didn't need to have its process checked", "${index}"),
         ifvarclass => "!start_process_${index}.!stop_process_${index}";
 
-    # On windows, we don't use processes promises type to check if process are running, as we rely directly on services
-    !windows.pass3::
```
Comment (amousset) [inline techniques/systemSettings/process/servicesManagement/3.0/servicesManagement.st]: this one should be kept I think

**Comment by amousset** [comment]
:rocket: :rocket: :rocket:

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
PR rebased

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
PR rebased

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
PR rebased

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**